pymc3/distributions/continuous.py

```diff
@@ -2478,7 +2481,17 @@ def logcdf(self, value):
         """
         alpha = self.alpha
         beta = self.beta
-        return bound(tt.log(tt.gammainc(alpha, beta * value)), value >= 0, alpha > 0, beta > 0)
+        # To avoid issue with #4340
+        safe_alpha = tt.switch(tt.lt(alpha, 0), 0, alpha)
```

Comment:
It is generally better practice to raise an Exception on invalid input (`ValueError`) than to silently change the input.

Reply:
I answered this in the main conversation.